todo_app/data/trello_items.py

```diff
@@ -0,0 +1,43 @@
+import os
+import requests
+
+trello_base_url = 'https://api.trello.com/1/'
+api_key = os.getenv('TRELLO_API_KEY')
+api_token = os.getenv('TRELLO_API_TOKEN')
+board_id = os.getenv('TRELLO_BOARD_ID')
+
+# TODO exercise-2: is there a nice way to not hard code
+todo_list_id = '6305fd4bf77620537efc5cf5'
+doing_list_id = '63063ebcfa6c6b7b70deac42'
+done_list_id = '6305fd4c6de4d31d4d1a97b3'
```

Comment:
Most people just make them environment variables, but you can try and figure them out dynamically at start up if you want